llama_index/vector_stores/google/generativeai/genai_extension.py

```diff
@@ -137,20 +137,32 @@ class Config:
         user_agent: The user agent to use for logging.
         page_size: For paging RPCs, how many entities to return per RPC.
         testing: Are the unit tests running?
+        google_auth_credentials: For setting credentials such as using service accounts
     """
 
     api_endpoint: str = _DEFAULT_API_ENDPOINT
     user_agent: str = _USER_AGENT
     page_size: int = _DEFAULT_PAGE_SIZE
     testing: bool = False
+    google_auth_credentials: Optional[credentials.Credentials] = None
```

Comment:
We can simplify `google_auth_credentials` to just `credentials`. Less verbose that way.